auditlog/models/rule.py

```diff
@@ -253,7 +253,7 @@ def read(self, fields=None, load='_classic_read', **kwargs):
                 return result
             self = self.with_context(auditlog_disabled=True)
             rule_model = self.env['auditlog.rule']
-            rule_model.create_logs(
+            rule_model.sudo().create_logs(
```

Comment:
@YogeshMahera-SerpentCS Your PR to OCA already had this here https://github.com/OCA/server-tools/pull/1830/files#diff-e2cdd2f4b9da04f93936a15fa53fef79R258 and how did we miss it in this PR?